src/ripple/app/misc/SHAMapStoreImp.h

```diff
@@ -106,8 +107,9 @@ class SHAMapStoreImp : public SHAMapStore
     std::uint32_t deleteInterval_ = 0;
     bool advisoryDelete_ = false;
     std::uint32_t deleteBatch_ = 100;
-    std::uint32_t backOff_ = 100;
-    std::int32_t ageThreshold_ = 60;
+    std::chrono::milliseconds backOff_{100};
+    std::chrono::seconds ageThreshold_{60};
+    boost::optional<std::chrono::seconds> recoveryBuffer_{};
```

Comment:
When I see something called `xxxBuffer` I think of memory buffers. We're using "buffer" in a different sense here. Maybe cal lit "Margin"? Or something else?

We should also leave a brief comment explaining what this does. I know the others don't, but it would be good to add comment for the new fields at least.
Also, nit: No need for the extra `{}`

Reply:
`recoveryWaitTime_` and renamed the config setting to `recovery_wait_seconds`.